sdk/eventhubs/azure-messaging-eventhubs/migration-guide.md

```diff
@@ -1,205 +1,170 @@
+### Package names and namespaces
 
-### Converting core classes
+Group ids, artifact ids, and package names for the modern Azure client libraries for Java have changed. They follow the [Java SDK naming guidelines][GuidelinesJavaDesign]. Each will have the group id `com.azure`, an artifact id following the pattern `azure-[area]-[service]`, and the root package name `com.azure.[area].[Service]`. The legacy clients have a group id `com.microsoft.azure` and their package names followed the pattern `com.microsoft. Azure.[service]`. This provides a quick and accessible means to help understand, at a glance, whether you are using modern or legacy clients.
 
-Creation of producers or consumers is done through either [EventHubClientBuilder][EventHubClientBuilder] or
-[EventProcessorClientBuilder][EventProcessorClientBuilder]. Asynchronous clients are created by invoking
-`builder.build*AsyncClient()`. Synchronous clients are created by invoking `builder.build*Client()`.
+In Event Hubs, the modern client libraries have packages and namespaces that begin with `com.azure.messaging.eventhubs` and were released starting with version 5. The legacy client libraries have package names starting with `com.microsoft.azure.eventhubs` and a version of 3.x.x or below.
 
-| In v3 | Equivalent in v5 | Sample |
-|---|---|---|
-| `EventHubClient.createFromConnectionString()` | `var builder = new EventHubClientBuilder().connectionString();`<br/>then either `builder.buildProducerAsyncClient();` or<br/>`builder.consumerGroup().buildConsumerAsyncClient();` | [Publishing events][PublishEventsWithCustomMetadata], [Consuming events][ConsumeEvents] |
-| `EventHubClient.createWithAzureActiveDirectory()` | `var builder = new EventHubClientBuilder().tokenCredential();`<br/>then either `builder.buildProducerAsyncClient();` or<br/>`builder.consumerGroup().buildConsumerAsyncClient();` | [Publishing events with Azure AD][PublishEventsWithAzureIdentity] |
-| `EventProcessorHost.EventProcessorHostBuilder`<br/>`.newBuilder()` | `new EventProcessorClientBuilder().buildEventProcessorClient()` | [EventProcessorClient with Azure Storage Blobs][EventProcessorClientInstantiation] |
+### Client hierarchy and constructors
 
-### Sending events
+In the interest of simplifying the API surface, we've made three clients. One client is for producing events, [EventHubProducerAsyncClient][EventHubProducerAsyncClient] has async operations and [EventHubProducerClient][EventHubProducerClient] is the sync version. The other two clients are intended for reading events. [EventProcessorClient][EventProcessorClient] is what you would typically use to read events from all partitions with an optional, but recommended checkpointing feature. [EventHubConsumerAsyncClient][EventHubConsumerAsyncClient] is what you would use for lower-level control of [EventData][EventData] consumption from a single partition.
```

Comment:
Ok, this got more confusing for me :) I think the "three clients" and the mix of sync and async clients is messing me up. Can we try something like the below?

```suggestion
In the interest of simplifying the API surface, we've made separate clients for sending and receiving events:
- [EventHubProducerClient][EventHubProducerClient] is the sync client to send events. [EventHubProducerAsyncClient][EventHubProducerAsyncClient] is the corresponding async client.
- [EventProcessorClient][EventProcessorClient] is what you would typically use to read events from all partitions with an optional, but recommended checkpointing feature. 
- [EventHubConsumerClient][EventHubConsumerClient] is the sync client you would use for lower-level control of [EventData][EventData] consumption from a single partition. [EventHubConsumerAsyncClient][EventHubConsumerAsyncClient] is the corresponding async client.
```